policyengine/countries/uk/reform_parameters.yaml

```diff
@@ -13,6 +13,19 @@ reforms:
           summary: Impose an LVT of @
         tests:
           - decreases_net_income: true
+  carbon:
+    rate:
+      values:
+        2021-01-01: 0.00
+      metadata:
+        policyengine:
+          title: Carbon Tax
+          short_name: carbon_tax
+          description: The price of a tonne of carbon emissions.
+          max: 250
```

Comment:
I think this could be 500. CCL [estimates](https://citizensclimatelobby.uk/wp-content/uploads/2021/05/CFDmathsUK-1.pdf) that a 560/ton tax would eliminate emissions entirely.

Reply:
I'm actually going to leave out the carbon and CT reforms, as they just need a few changes in openfisca-uk.